### PR TITLE
defaultValue check for null

### PR DESCRIPTION
**Pull request by Rudraksha20**
Worked on issue #5511, defaultValue check for Null.

### CODE REVIEW

**Comment by pjcozzi** [comment]
Thanks @Rudraksha20!  Could you also please add a small note to the 1.35 section of [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CHANGES.md) to mention that `defaultValue` now also checks for `null`?

**Comment by Rudraksha20** [comment]
Updated CHANGES.md.

**Comment by hpinkos** [comment]
@mramato I feel like I remember having a discussion about this before.  Did you have a reason for not checking `null` in `defaultValue`?

**Comment by mramato** [comment]
At one point Cesium wanted to basically pretend `null` doesn't exist and if it showed up anywhere, then it must be a bug.  However, this proved foolish in practice because external data or DOM API calls end up giving us null values on a regular basis.  There were also performance concerns at one point with doing extra checks which have long since gone away now that browsers are so much faster.

So I'm okay with this change, but someone should smokescreen the widgets and some Sandcastle examples before merging this since it could potentially have unintended consequences (though I doubt we were relying on `defaultValue` returning `null` anywhere.

**Comment by pjcozzi** [comment]
> So I'm okay with this change, but someone should smokescreen the widgets and some Sandcastle examples before merging this since it could potentially have unintended consequences (though I doubt we were relying on defaultValue returning null anywhere.

@omh1280 could you please do this?

File: CHANGES.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ Change Log
 * Added a Sandcastle demo for setting time with the Clock API [#5457](https://github.com/AnalyticalGraphicsInc/cesium/pull/5457);
 * Added support for `ParticleSystem`s. [#5212](https://github.com/AnalyticalGraphicsInc/cesium/pull/5212)
 * Added `Cesium.Math.randomBetween`.
+* `deafultValue` updated to check for Null.
```
Comment (omh1280) [inline CHANGES.md]: Spelling!

Comment (omh1280) [inline CHANGES.md]: And link this PR in CHANGES.md

**Comment by omh1280** [comment]
Everything looks good to me. Thanks @Rudraksha20!

**Comment by Rudraksha20** [comment]
@omh1280 updated CHANGES.md.

**Comment by omh1280** [comment]
Looks good! One of the tests is failing (timeout), but it's not failing locally. This makes me think it's a Travis issue, not a code issue.

**Comment by hpinkos** [comment]
I think you're right @omh1280.  I restarted the travis build to see if it passes.

@Rudraksha20 can you please add a unit test?

**Comment by Rudraksha20** [comment]
@hpinkos, added Unit test for defaultValue.

File: Specs/Core/defaultValueSpec.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+/*global defineSuite*/
+defineSuite([
+        'Core/defaultValue'
+    ], function(
+        defaultValue) {
+    'use strict';
+
+    it('Works with first parameter undefined', function() {
+        expect(function(){
+            defaultValue.defaultValue(undefined, 5);
```
Comment (omh1280) [inline Specs/Core/defaultValueSpec.js]: This can just become `expect(defaultValue(undefined,5).toEqual(5)`, I think

Comment (Rudraksha20) [inline Specs/Core/defaultValueSpec.js]: @omh1280 yup, updated with the changes.

**Comment by omh1280** [comment]
Should be good once this goes green!

**Comment by hpinkos** [comment]
Great work @Rudraksha20!  I just tweaked the working in `CHANGES.md `
Thanks for reviewing @omh1280!

**Pull request merged**